pkgs/development/compilers/gcc/builder.sh

```diff
@@ -203,6 +203,17 @@ preInstall() {
         ln -s lib "$out/${targetConfig}/lib32"
         ln -s lib "${!outputLib}/${targetConfig}/lib32"
     fi
+
```

Comment:
We should migrate this eventually to the normal nix phases and share those between all gcc.

Same with a good chunk of the other settings, they are often just copied between all files. I think there was a PR that did that recently.